HCCGo/app/html/clusterLanding.html

```diff
@@ -63,12 +84,19 @@ <h4 class="usage text-center">/work Usage</h4>
               <i class="mdi-action-trending-neutral" ng-if="job.running" />
               <i class="mdi-action-query-builder" ng-if="job.idle" />
               <i class="mdi-alert-error" ng-if="job.error" />
-              <i class="glyphicon glyphicon-check" ng-if="job.complete && !job.error" />
+              <i class="glyphicon glyphicon-check" ng-if="job.complete && !job.error && !job.cancelled" />
+              <i class="glyphicon glyphicon-ban-circle" ng-if="job.cancelled"/>
+              
             </div>
             <div class="row-content">
               <div class="action-secondary">
-                <a class="deleteButton" href="javascript:void(0)" ng-click="removeCompletedJob(job._id, $event)" ng-if="job.complete">
+                <button type="button" class="btn btn-xs deleteButton" ng-click="removeCompletedJob(job._id, $event)" ng-if="job.complete">
                   <i class="glyphicon glyphicon-remove"></i>
+                  <div>Remove</div>
+                </a>
+                <a class="cancelButton" href="javascript:void(0)" ng-click="cancelModal(job, $event)"  ng-if="job.running || job.idle">
```

Comment:
This was me just testing one, since it will basically be copy and paste from there. I'll change it once I get the first one all good.